pkg/api/api.proto

```diff
@@ -144,12 +144,15 @@ service OverviewService {
 }
 
 message GetOverviewRequest {
+  // Retrieve the overview at a certain state of the repository. If it's empty, the latest commit will be used.
+  string git_revision = 1;
```

Comment:
```suggestion
  string gitRevision = 1;
```

Reply:
buf and the official protobuf style guide recommend lower snake case for fields: https://protobuf.dev/programming-guides/style/#message-field-names
we mostly follow this and I would rather go with the style guide and fix the remaining occurences where we don't

Reply:
Ok, then please change the api.proto accordingly.

Reply:
Or plan a story if it's taking a lot of time. 
Ideally we should also find a linter for this and enable it in the CI

Reply:
I already have a story for the whole buf lint suite. That would cover this as well.